roles/middleware/files/FWO.Middleware.Server/Program.cs

```diff
@@ -26,7 +26,21 @@
 APIConnection apiConnection = new APIConnection(configFile.ApiServerUri ?? throw new Exception("Missing api server url on startup."), jwtWriter.CreateJWTMiddlewareServer());
 
 // Fetch all connectedLdaps via API (blocking).
-List<Ldap> connectedLdaps = apiConnection.SendQueryAsync<List<Ldap>>(AuthQueries.getLdapConnections).Result;
+List<Ldap> connectedLdaps = new List<Ldap>();
+while (true)
+{
+    // Repeat first api call in case graphql api is not started yet
+    try
+    {
+        connectedLdaps = apiConnection.SendQueryAsync<List<Ldap>>(AuthQueries.getLdapConnections).Result;
+        break;
+    }
+    catch (Exception ex)
+    {
+        Log.WriteError("Graphql api", "Graphql api unreachable.", ex);
```

Comment:
Maybe we could put a little wait (maybe increasing with numbers of attempts) here not to blaster the log files?